### PR TITLE
2023 06 06 default fgal dir

### PR DESCRIPTION
**Pull request by tiki-jyhem**
## Problem

- The instructions for setting the files in ADMIN.md displayed « Please use the preset path directory: /home/yunohost.app/$app » instead of replacing $app with "Tiki" or "Tiki__1" etc. The proper value for $app changes when one installs more than one Tiki.

## Solution

- I patch the Tiki code so as to make the correct Yunohost path the default Tiki path.
- Users don't need to know and cut-paste the correct path any more.

## PR Status

- [ x] Code finished and ready to be reviewed/tested
- [ x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: doc/ADMIN.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 To ease the install process and first access, Tiki saves your uploaded files (office documents, images, pdf, etc. attached to wiki pages, forum posts, tracker items, file galleries...) by default in its database. This works perfectly in most cases but it is not the recommended setup if you need to save many thousands of files or more.
 
-In that case, consider switching from "Store to database" to "Store to directory", which you will find in the Configuration Wizard. Please use this preset path directory: `/home/yunohost.app/$app`. You will be able to migrate your currently uploaded files from one to the other.
+In that case, consider switching from "Store to database" to "Store to directory", which you will find in the Configuration Wizard. Please use the preset path directory: `/home/yunohost.app/tiki<…>`. You will be able to migrate your currently uploaded files from one to the other.
```
Comment (ericgaspar) [inline doc/ADMIN.md]:
```suggestion
In that case, consider switching from "Store to database" to "Store to directory", which you will find in the Configuration Wizard. Please use the preset path directory: `__DATA_DIR__`. You will be able to migrate your currently uploaded files from one to the other.
```

Comment (ericgaspar) [reply doc/ADMIN.md]: with app v2 format, we are now able to get the proper path with `__DATA_DIR__` set in `ADMIN.md`

File: scripts/install [hunk 2]
```diff
@@ -71,14 +71,22 @@ exec_as_app console.php database:configure "$app" "$db_pwd" "$app"
 # Create database contents
 exec_as_app console.php database:install
 
+# Set default database directory
+FGALDIR=/home/yunohost.app/$app
```
Comment (ericgaspar) [inline scripts/install]: there is already a variable name for `/home/yunohost.app/$app` path:
`$data_dir="/home/yunohost.app/$app"`

File: scripts/install [hunk 3]
```diff
@@ -71,14 +71,22 @@ exec_as_app console.php database:configure "$app" "$db_pwd" "$app"
...
+cd "$install_dir"
+sed -i -e "s#storage\/fgal#$FGALDIR#" lib/prefs/fgal.php
+
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
# Set default database directory
cd "$install_dir"
sed -i -e "s#storage\/fgal#$data_dir#" lib/prefs/fgal.php
```

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**